hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/TargetStatusForDistributionSetEvent.java

```diff
@@ -0,0 +1,62 @@
+package org.eclipse.hawkbit.repository.model;
```

Comment:
Moved to org.eclipse.hawkbit.repository.event.remote package. This contains other events like DownloadProgressEvent, RolloutDeletedEvent etc. Thought this would be a good place.